libs/shared/ui-components/src/Form/Form.tsx

```diff
@@ -35,6 +36,7 @@ export const Form: FC<TFormProps> = (props) => {
   };
 
   const onSubmit = handleSubmit((formValues): void => {
+    gtag_report_conversion();
```

Comment:
Pretty sure the function is expecting to receive the current page URL here. Not sure if it's as simple as adding `window.location.href`?

```suggestion
    gtag_report_conversion(window.location.href);
```